### PR TITLE
Rename extension - Part 1 - New name

### PR DESCRIPTION
**Pull request by gadenbuie**
Updates the extension ID to be `Posit.shiny` and moves the extension to `v1.0.0`.

The outdated extension points to this extension as a dependency and uninstalls itself upon activation.

For #53

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,11 +20,9 @@
     "Other"
   ],
   "activationEvents": [
+    "onStartupFinished",
     "onLanguage:python",
-    "onCommand:shiny.python.runApp",
-    "onCommand:shiny.python.debugApp",
-    "onLanguage:r",
-    "onCommand:shiny.r.runApp"
+    "onLanguage:r"
```
Comment (gadenbuie) [inline package.json:25]: Because we are now activating `onStartupFinished` and `onLanguage:python` or `onLanguage:r`, I don't think we need the `onCommand` activation events

File: src/shinylive.ts [hunk 2]
```diff
@@ -542,12 +542,10 @@ function shinyliveUrlDecode(url: string): ShinyliveBundle | undefined {
...
   const pathParts = pathname.split("/");
+  const language: ShinyliveLanguage = pathParts.includes("py") ? "py" : "r";
+  const mode: ShinyliveMode = pathParts.includes("editor") ? "editor" : "app";
 
-  return {
-    language: pathParts[1] as ShinyliveLanguage,
-    files: files as ShinyliveFile[],
-    mode: pathParts[2] as ShinyliveMode,
-  };
+  return { language, mode, files: files as ShinyliveFile[] };
```
Comment (gadenbuie) [inline src/shinylive.ts:548]: Okay, technically this is an unrelated change and normally I'd do it in a separate PR. But this is small, easy, and actually has no impact other than making sure I don't forget. (We don't actually use `language` or `mode`, we only need `files` downstream of this function.)

**Comment by gadenbuie** [comment]
Now that the old extension handles removing itself, if users install `Posit.shiny` with the current version of `Posit.shiny-python`, they won't see errors but the `Posit.shiny-python` extension will likely fail to load. (At least that's what happened for me and I'm hoping it's an alphabetical sorting that decides which one wins.)

If users even realize that there's an issue, they could go to the old extension's setting page where they would see the page below. That said, once we send out the shell `Posit.shiny-python` extension they'd also be able to update from this screen, eventually leading to the old extension removing itself.

image

File: src/shinylive.ts [hunk 3]
```diff
@@ -181,7 +181,7 @@ export async function shinyliveSaveAppFromUrl(): Promise<void> {
 
   const doc = await vscode.workspace.openTextDocument(localFiles[0]);
 
-  await vscode.window.showTextDocument(doc, 2, false);
+  await vscode.window.showTextDocument(doc, undefined, false);
```
Comment (gadenbuie) [inline src/shinylive.ts:184]: Another small quality of life change that's worth getting in: when we open the primary file of a Shinylive app after saving it locally, we now open the file in the current column (instead of the 2nd column).

**Review by jcheng5 — APPROVED** [review]
